backend/test/emr/test_create_emr_cluster.py

```diff
@@ -98,13 +99,16 @@ def _expected_args(custom_ami: Optional[str] = None):
         "LogUri": "s3://mlspace-log-bucket",
         "ReleaseLabel": MOCK_RELEASE_LABEL,
         "Applications": [{"Name": "Hadoop"}, {"Name": "Spark"}],
+        "ReleaseLabel": MOCK_RELEASE_LABEL,
+        "Applications": [{"Name": "Hadoop"}, {"Name": "Spark"}],
```

Comment:
Are these lines really meant to be repeated? Same for InstanceType below.

Reply:
I did a merge from main and for some reason it did this in a bunch of places, looks like I missed some. Thanks for catching it!